### ISSUE TITLE
The filter dropdown in igGrid should announce when the dropdown menu is expanded or closed

### ISSUE THREAD

**Issue by kchanda24** [IgniteUI/ignite-ui#2241]
## Description  
We are consumers of IgGrid. During a recent evaluation by a third-party accessibility vendor, it we found out that the state of the filter dropdown button is not announced when it is expanded or collapsed. This is required in accordance with WCAG 2.0 Level A guidelines.

## Version and Browser used :

ignite-ui version: any
browser: any

## Steps to reproduce  

Go to this [sample](https://jsfiddle.net/georgianastasov/kn7ru52s/2/).
Use NVDA or other accessibility tool.
Reach the filter buttons in the grid.
Expand the filter options dropdown menu.

## Result  
The screen reader does not announce the state change when the dropdown is expanded or collapsed.

## Expected result  
The screen reader should announce the state changes.

## Attachment
![filterExpansionNotAnnounced](https://github.com/IgniteUI/ignite-ui/assets/157054087/89cf5dc5-911a-4079-93a1-a14df09f9521)

**Comment by ChronosSF** [comment]
The issue has been fixed and will be released early next week.
